### DISCUSSION TITLE
Domain Issue

### DISCUSSION THREAD

**Discussion by Ayhamjabak** [emailjs-com/help#62] [Q&A] [answered]
I lunched the application locally on my pc, i recieve emails without any error, but after building my application and lunching it on my domain i am not recieving emailsany more. But locally it is working perfectm any help please would appreciate it.

Thanks in advance.

**Comment by xr0master** [comment, marked as answer]
Most likely, you are using environment variables, and they are not configured correctly for production.

**Reply by Ayhamjabak** [reply]
thanks issue is solved
